### ISSUE TITLE
Summarize problem specification

### ISSUE THREAD

**Issue by dkubek** [dkubek/mcf-simplex-analyzer#7]

**Comment by martinkoutecky** [comment]
Does the [hackmd description](https://hackmd.io/Uut8UzwwQgutnzPZMxNh9Q) resolve this?

**Comment by dkubek** [comment]
> Does the [hackmd description](https://hackmd.io/Uut8UzwwQgutnzPZMxNh9Q) resolve this?

Yes it does, I was just thinking of somehow making it also available in the repository.
